### ISSUE TITLE
UnhandledPromiseRejectionWarning about colors.json when compiling to react

### ISSUE THREAD

**Issue by lyahdav** [airbnb/Lona#37]
```
➜ node index.js LonaWorkspace1.component output
(node:71950) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOTDIR: not a directory, open '<snip>/LonaWorkspace1.component/colors.json'
(node:71950) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

I was thinking maybe if I use some colors it would resolve this, but if I try to set a custom color on a View's border, it doesn't seem to work yet (the Custom colors drop down is empty).

**Comment by dabbott** [comment]
Hey man! Hmm... ok, firstly, I haven't touched the code gen script in a bit since I'm working on a better one. But I think 

1. you'll want to pass a directory as the first argument, rather than a component, e.g. `node index.js LonaWorkspace output`
2. if the custom colors aren't showing, that probably means (1) the workspace hasn't been set in preferences, or (2) there isn't a `colors.json` in the workspace directory. 

If you try the material design workspace example in the `examples` folder in the repo, does that show custom colors?

**Comment by lyahdav** [comment]
Hey Devin! Yeah, this is my bad. Was just documenting issues as I ran into them and was going to document how I resolved, didn't expect such a fast response. I resolved with a few changes:

1. As you said I needed to pass in the workspace path, which I first configured in preferences. Then I moved my `.component` into that workspace path.
1. I had to manually create `colors.json` and `textStyles.json` in the workspace. For now those files just include an empty JSON object `{}`.
